common/src/test/java/org/apache/sedona/common/FunctionsTest.java

```diff
@@ -2861,6 +2862,44 @@ public void locateAlong() throws ParseException {
         assertEquals("Polygon geometry type not supported, supported types are: (Multi)Point and (Multi)LineString.", e.getMessage());
     }
 
+    @Test
+    public void isValidDetail() throws ParseException {
+        // Valid geometry
+        Geometry validGeom = GEOMETRY_FACTORY.createPolygon(coordArray(30, 10, 40, 40, 20, 40, 10, 20, 30, 10));
+        ValidDetail actualValidDetail = Functions.isValidDetail(validGeom);
+        ValidDetail expectedValidDetail = new ValidDetail(true, null, null);
+        assertTrue(expectedValidDetail.equals(actualValidDetail));
+
+        Integer OGC_SFS_VALIDITY = 0;
+        Integer ESRI_VALIDITY = 1;
+
+        actualValidDetail = Functions.isValidDetail(validGeom, OGC_SFS_VALIDITY);
```

Comment:
add a test case when any property is null?

Reply:
What do you mean?